odoo-shippable/scripts/build-image.sh

```diff
@@ -51,7 +51,8 @@ DPKG_DEPENDS="postgresql-9.3 postgresql-contrib-9.3 postgresql-9.5 postgresql-co
 PIP_OPTS="--upgrade \
           --no-cache-dir"
 PIP_DEPENDS_EXTRA="line-profiler watchdog coveralls diff-highlight \
-                   pg-activity virtualenv nodeenv setuptools==33.1.1"
+                   pg-activity virtualenv nodeenv setuptools==33.1.1 \
+                   html2text==2016.9.19"
```

Comment:
@ruiztulio What about use a wget of the requirements.txt and delete the unused lines with sed and install this file?